dali/pipeline/operator/builtin/external_source.cc

```diff
@@ -81,6 +99,20 @@ where the last dimension represents the different channels).)code")
   .NumOutput(1)
   .AddOptionalArg("blocking",
       R"code(Whether external source should block until data is available or just
-fail when it is not)code", false);
+fail when it is not)code", false)
+  .AddOptionalArg("no_copy",
+      R"code(If DALI should copy the buffer when feed_input is called
+If true, then instead of making a copy of the provided buffer,
```

Comment:
same comments can be applied here and in the python documentation

Reply:
Done